src/modules/dashboard/components/ActionsPage/MultisigWidget/MultisigWidget.css

```diff
@@ -0,0 +1,58 @@
+.wrapper {
+  padding: 10px 0;
+}
+
+.status {
+  display: block;
+  margin-top: -6px;
+  height: 7px;
+  width: 7px;
+  position: relative;
+  float: right;
+  border-radius: 50%;
+  background-color: rgb(255, 176, 0, 1);
```

Comment:
Arg! you used the `rgba` notation while declaring a `rgb` color. Please remove the trailing `1`

Also, please set this inside a `@value`